x-pack/legacy/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -48,6 +48,24 @@ function isLocalStateDirty(
   );
 }
 
+// Exported for testing purposes, sets the Kibana breadcrumbs.
+export function setBreadcrumbs(
+  chrome: {
+    addBasePath: (url: string) => string;
+    breadcrumbs: { set: (crumbs: Array<{ text: string; href?: string }>) => void };
+  },
+  persistedDoc?: { title: string }
+) {
+  const rootCrumb = {
+    href: chrome.addBasePath(`/app/kibana#/visualize`),
+    text: i18n.translate('xpack.lens.breadcrumbsTitle', {
+      defaultMessage: 'Visualize',
+    }),
+  };
+
+  chrome.breadcrumbs.set(persistedDoc ? [rootCrumb, { text: persistedDoc.title }] : [rootCrumb]);
```

Comment:
I don't feel super strong about this, but in Visualize there is a "Create" crumb if the current is not saved:
<img width="199" alt="Screenshot 2019-08-30 at 09 36 03" src="https://user-images.githubusercontent.com/1508364/64002143-a7b03d00-cb09-11e9-8fc3-0d66cc19c78d.png">

Feels like we should do the same thing